script/Arcueid/Day 8/08_00_ARC08_6.txt

```diff
@@ -197,62 +214,74 @@ I think I'm going crazy, but I can't figure out what's wrong with my body at all
 [sha:72daa7ea1fd433ebb3987f3a340d82cba8f523d4]{
 -- Page 305, Offset 20428.
 -- それを言われると、返答に困る。
-It's hard to respond to her when she says it like that.
+I don't have an easy answer to a statement like that.
 }
 [sha:dc96a1819915a0104dabe54c9176fbf85b7d3da9]{
 -- Page 305, Offset 20429.
 -- アルクェイドを十七個に解体してしまった手前、身体に異常があるといわれたら、こっちはもう平謝りするしかない。
-I dismantled Arcueid's body into 17 pieces, and if I was told that her body was messed up as a result, I would have no other choice but to apologize sincerely.
+I'm the bastard responsible for dismantling Arcueid's body into seventeen different pieces.
+ If she tells me that something's now wrong with it as a result,
+ what can I do but bow my head and apologize.
 }
 [sha:67fc4cc18361ea62cd3318ee00affb56ccc5d994]{
 -- Page 306, Offset 20430. Mods: @n.
 -- 「ま、いいわ。
-"Well enough with that.
+"Well, never mind all that.
 }
 [sha:5e0b110b5b9cb94b18a4df9d6bac12b61d5acbaf]{
 -- Page 306, Offset 20431. Glued.
 -- 時間もなくなっちゃったし、無駄話をしている暇なんてないものね」
-We don't have much time, so we can't afford to be idly chatting around."
+// Leading space for glue
+ Time's a wasting, we can't just stand here shooting the breeze all night."
```

Comment:
I think this is too much. Revert imo

Reply:
Agree that time's a wasting is pushing it, but can we keep shooting the breeze? "idly chatting around" doesn't really parse to me

Reply:
yeah, that parts ok